Add Jamorphosia

0vordhosbn00 opened the pull request:
Add Jamorphosia project

yousefebrahimi0 commented:
Hi, regarding to your question:
Jamorphosia!
-The lack of specific information from the manufacturer or the team behind the website and the absence of any email or clear usual communication method and...
-Lack of privacy policy and details on data collection/usage. The terms state user data is not shared with third parties, but there are no details provided on exactly what data is collected and how it is used/stored. This is problematic from a privacy perspective.
-No mention of copyright/DMCA policies. The terms tell users not to upload copyrighted content, but provide no information on how copyright infringement claims will be handled.

0vordhosbn00 commented:
Hello Yousef,

Following your latest recommendations, I made some changes to my website:
- Added a contact email address on the "Legal Notice" and "General conditions of sale and use" page.
- Added information on the data collected, and the purpose of the collection ("Information regarding data collected" part of the "General conditions of sale and use" page).
- Added copyright information ("Respect of copyright", "User Responsibility" and "Copyright Infringement Notice" sections of the "General conditions of sale and use" page).


Regarding your remark "The lack of specific information from the manufacturer or the team behind the website": I am the only developer of the website, and it is indicated that the website is a creation of the company "Netpresso". I don't know what to add on this subject.

Regarding your remark "The terms tell users not to upload copyrighted content, but provide no information on how copyright infringement claims will be handled": The generated files are only visible to the users who created them, and no one else can see them. It is therefore impossible for a person to notice a copyright infringement if they have not done so themselves.

Do you think my site could now be included in your list? In any case, thank you for helping me improve my website.

Thank you in advance, I wish you a good day.
